### PR TITLE
Modified the text css

### LINKED ISSUE #454: Text in white on home page not quite visible
# 🐞 Bug Report

### **Describe the bug**
"Station Saarthi: Your platform guide" is written in white color. 

### **Steps to Reproduce**
1. Go to website
2. See error on home page.

### **Expected behavior**
To be more visible, it should be written in black.

### **Actual behavior**
The text is written in white which is not looking good at a light background.

### **Screenshots or Video**
image

### PR DESCRIPTION
**Pull request by shreya2k3**
# Pull Request Template for GSoC 2024

## Related Issue

#454 

closes: #454 

## Description of Changes

A background has been added to the white text so that it is more readable

## Type of Pull Request

- [X] Bug Fix
- [X] Performance Improvement

(Please check relevant boxes by entering an "X" inside them)

## Screenshots / Demo (if applicable)
image

## Checklist for Contributor
use'X' in [ ] to check it .

- [X] I have reviewed my code before submission.
- [X] I have read and followed the project's Contribution Guidelines.
- [X] I have tested the code thoroughly.
- [X] I have attached related issue numbers, screenshots, or videos (if applicable).
- [X] I have added comments to my code where necessary for clarity.
- [X] I have checked the existing code base and my changes do not introduce any breaking changes.
- [X] My PR addresses only one issue; I have created separate PRs for other issues.
- [X] I have ensured that the PR does not include unrelated files.
- [X] I am mindful of the project's coding standards and best practices.

## Additional Information

[Include any additional context or relevant information that may help in reviewing this PR.]

## Contribution Guidelines Reminder

Thank you for contributing to our project! Please ensure you follow the guidelines below:

### Contribution Levels

2. **Task Categories:**
   - **Basic Tasks:** Primarily focused on small changes like minor bug fixes, documentation improvements, and unit testing. Labeled as **Easy**.
   - **Intermediate Tasks:** Moderate-level tasks that involve enhancing existing features, code refactoring, or performance improvements. Labeled as **Medium**.
   - **Advanced Tasks:** Complex tasks like building new features, significant refactoring, or critical bug fixes. Labeled as **Hard**.

3. **Acknowledging Effort:**
   - Exceptional contributions will be recognized with a higher contribution level if substantial effort and innovation are demonstrated.

### Review Process

- PRs will be reviewed by the maintainers, and feedback will be provided. Contributors should address feedback promptly and resubmit the PR if necessary.

- Contributors are encouraged to participate in the review process of other PRs for community growth.

We appreciate your time and effort in improving this project and look forward to your valuable contributions!

### CODE REVIEW

**Comment by dhairyagothi** [comment]
@shreya2k3 do it in namaste yatree also 
and insstead of black use light blue

File: frontend/src/Pages/Herosection.jsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ const Herosection = () => {
       <div className="relative flex items-center justify-center bg-center bg-cover herosection">
         <div className="relative z-10 grid justify-items-center">
           <img src={logo} alt="" srcset="" style={{ height: "40vh" }} />
-          <h1 className="text-2xl font-extrabold text-white text-center">
+          <h1 className="text-2xl font-extrabold text-white text-center bg-black bg-opacity-50 px-4 py-2 rounded-lg">
             Station Saarthi : Your Platform Guide
```
Comment (dhairyagothi) [inline frontend/src/Pages/Herosection.jsx]: @shreya2k3 do it in 

> namaste yatree

 also 
and insstead of black use light blue

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

File: frontend/src/Pages/Herosection.jsx (hunk 1 above)
Comment (shreya2k3) [reply frontend/src/Pages/Herosection.jsx]: Done 👍

**Comment by dhairyagothi** [comment]
image

i don;'t think it is good looking

**Comment by shreya2k3** [comment]
But it is looking better, earlier it was not even readable

**Comment by dhairyagothi** [comment]
> But it is looking better, earlier it was not even readable

just change font colour to dark blue and remove bg colour

**Pull request closed without merging**